coding/coding_tests/bit_streams_test.cpp

```diff
@@ -19,19 +19,19 @@ namespace
 {
 UNIT_TEST(BitStreams_Smoke)
 {
-  uniform_int_distribution<uint8_t> randomBytesDistribution(0, 255);
+  uniform_int_distribution<uint32_t> randomBytesDistribution(0, 255);
```

Comment:
Please, show the compiler output here, what is wrong with uint8_t.

Reply:
It is included in the commit message:
> error C2338: static_assert failed: 'invalid template argument for uniform_int_distribution: N4950 [rand.req.genl]/1.5 requires one of short, int, long, long long, unsigned short, unsigned int, unsigned long, or unsigned long long'